Resources/Prototypes/_NF/Entities/Objects/Devices/holoprojectors.yml

```
@@ -24,3 +24,31 @@
     slots:
       cell_slot:
         name: power-cell-slot-component-slot-name-default
+
+- type: entity
+  parent: Holoprojector
+  id: HoloGraffitiProjector
+  name: holograffiti projector
+  description: Stop suicidal passengers from killing everyone during atmos emergencies.
```

Could use a rewrite, here's a suggestion, but I'm out of my element.
```suggestion
  description: Tag up the sharp end, let 'em know you were here. 
```